Remotion/Web/Development.WebTesting/HostingStrategies/IisExpressProcessWrapper.cs

```diff
@@ -72,7 +71,8 @@ public void Dispose ()
     {
       if (_iisProcess != null && !_iisProcess.HasExited)
       {
-        _iisProcess.CloseMainWindow();
+        _iisProcess.Kill();
+        _iisProcess.WaitForExit();

```

Comment:
Should we have a timeout here? And sensible handling in case the timeout is exceeded.

Reply:
Added timeout in fdf24412d8415c9e648b5e4288de06b3811d9323. I'm not sure about the exception type...